City-of-Pori/pori-kada
# PORI-117: Text size block removal

Text size block hidden for the time being.

To test:
1. drush fra -y && drush cc all
2. go to local site and confirm that text size block is not present next to language-switcher.
